modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/ObjectFieldFormBase.tsx

```diff
@@ -122,7 +125,7 @@ async function getFieldSettingsByBusinessType(
 	const {businessType, objectFieldSettings} = values;
 
 	if (businessType === 'Picklist' || businessType === 'MultiselectPicklist') {
```

Comment:
@igor-franca sort the condition like `if (businessType === 'MultiselectPicklist' || businessType === 'Picklist') {`